products/jenkins.md

```diff
@@ -15,9 +15,10 @@ auto:
 
 # releaseCycle 2 catches weekly releases, other are LTS.
 # For an LTS :
-# - Its support date is its release date, meaning there will never be backport of features.
+# - Its end of support date is the 'x.y.1' release date. Most of the new features backported in LTS
```

Comment:
@MarkEWaite, is this new comment better than the old one ?

Reply:
```suggestion
# - Its support begins with the 'x.y.1' release date. The `supportDate` value should use that date.  Most of the new features backported in LTS
```

Isn't that date the beginning of support, not the end?

Reply:
No, it's the end of support date (the naming is confusing). I thought it was OK to "end" the  considering after the 'x.y.1' release most of the changes are fixes.

But looking at https://endoflife.date/jenkins I realize that the `support` fields must be removed, because the `activeSupportColumn` was set to `false` in #2511, so those dates are never displayed.